sketch.js

```diff
@@ -78,41 +78,47 @@ const sketch = ({ context }) => {
   const earthMesh = new THREE.Mesh(geometry, earthMaterial);
   createPlanet(scene, earthMesh, earthGroup, 31, 1);
 
+  const moonTexture = loader.load("textures/moon.jpg");
+  const moonMaterial = new THREE.MeshPhongMaterial({ map: moonTexture});
+  const moonGroup = new THREE.Group();
+  const moonMesh = new THREE.Mesh(geometry, moonMaterial);
+  createPlanet(earthGroup, moonMesh, moonGroup, 33, 0.3);
```

Comment:
aici o sa fie o problema cu partea de shadows pe care am adaugat-o in celalalt PR. poti sa schimbi din createPlanet(....) in 
` moonMesh.position.set(33, 0, 0);
  moonMesh.scale.setScalar(0.3);
  moonGroup.add(moonMesh);
  earthGroup.add(moonGroup);`